backend/homework/1/app/src/main/java/kdu/backend/studManage/Main.java

```diff
@@ -0,0 +1,84 @@
+package kdu.backend.studManage;
+import java.util.Scanner;
+
+public class Main {
+    static Scanner sc = new Scanner(System.in);
+
+    public static void main(String[] args) {
+        StudentRepository sr = new StudentRepository();
+
+        System.out.println("Choose Options: ");
+        System.out.println("1. Add Students");
+        System.out.println("2. Update Students");
+        System.out.println("3. Retrieve Student by Id");
+        System.out.println("4. Retrieve Student by Name");
+
+        int op = 1;
+
+        while(op==1){
+
+            int choice = sc.nextInt();
+            switch(choice) {
+                case 1:
+                    option1(sr);
+                    break;
+                case 2:
+                    option2(sr);
+                    break;
+                case 3:
+                    option3(sr);
+                    break;
+                case 4:
+                    option4(sr);
+                    break;
+                case 5:
```

Comment:
What is the 5th choice? Please write an exit choice logic and invalid choice logic